### PR TITLE
Add initial text and image to new portfolio page

### PR DESCRIPTION
**Pull request by lilymzhou**
- updated default README to describe new portfolio page
- Add paragraph introduction to main page
- Add picture to main page
- Style: Fix indentation in index.html

Screenshot 2020-06-22 at 12 18 23 PM

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
+    <title>Lily Zhou</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>Hi! I'm Lily, a STEP Intern at Google this summer and a rising junior @ Stanford studying CS + English.</p>
+    <img src="/images/flower.jpg" alt="Red Flower">
```
Comment (megankj) [inline portfolio/src/main/webapp/index.html:13]: align the img tag with the surrounding p tags

**Review by megankj — COMMENTED** [review]
In the description could you also include a screenshot of what it now looks like?

**Comment by lilymzhou** [comment]
Screenshot 2020-06-22 at 12 18 23 PM

**Comment by megankj** [comment]
A small nit: the PR title is "Update the README.md file to describe the repo" but this change also includes changes to index.html and uploading an image.

I would modify the title of the PR to more accurately reflect the changes and update the description to include all the changes being made (I know the individual commits include what was changed, but I still think it's good practice to write it out!).

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+    <!DOCTYPE html>
```
Comment (tjgoog) [inline portfolio/src/main/webapp/index.html]: Nit: It looks like the whole file has had an extra level of indentation added, which I think should be removed.

**Review by tjgoog — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (megankj) [reply portfolio/src/main/webapp/index.html]: To clarify, there should be two spaces of indentation.

**Review by megankj — APPROVED** [review]
thanks for making the changes!

**Review by tjgoog — APPROVED** [review]
Looks Good To Me!

File: README.md [hunk 3]
```diff
@@ -1,13 +1,3 @@
 # Google Student Training in Engineering Program
```
Comment (tjgoog) [inline README.md:1]: Sorry I didn't notice this earlier-- I think you should change this title line to be more specific, e.g. "Lily's STEP Portfolio" or something, or just remove this line and the following blank line.

**Pull request merged**